src/tests/index.test.js

```diff
@@ -0,0 +1,326 @@
+
+import * as assert from 'assert';
+
+import EvolvAssetManager from '../index.js';
+import { DocumentMock, StyleSheetMock, ScriptMock } from './mocks/document.mock.js';
+import EvolvMock from './mocks/evolv.mock.js';
+
+function generateJsVariants(invokedJavascript) {
+	return {
+		'evolv_web_page1': () => {
+			return new Promise((resolve, reject) => {
+				invokedJavascript.push('evolv_web_page1');
+				resolve(true);
+			});
+		},
+		'evolv_web_page1_variable1': () => {
+			return new Promise((resolve, reject) => {
+				invokedJavascript.push('evolv_web_page1_variable1');
+				resolve(true);
+			});
+		},
+		'evolv_web_page1_variable2': () => {
+			return new Promise((resolve, reject) => {
+				invokedJavascript.push('evolv_web_page1_variable2');
+				resolve(true);
+			});
+		}
+	}
+}
+
+describe('asset manager handles correctly', () => {
+	const evolvCssAssetSrc = 'https://participants-test.evolv.ai/v1/testenv/test-uid/assets.css';
+	const evolvJsAssetSrc = 'https://participants-test.evolv.ai/v1/testenv/test-uid/assets.js';
+	const options = {
+		user: {
+			uid: 'test-uid',
+			sid: 'test-sid'
+		}
+	}
+
+	describe('given no assets on page', () => {
+		global.window = {location: {href: 'https://test-site.com'}, _evolv: {}};
```

Comment:
You renamed this in another PR, right?
```suggestion
		global.window = {location: {href: 'https://test-site.com'}, evolv: {}};
```

Reply:
Yah